infra/bitcoin/run.sh

```diff
@@ -28,6 +29,6 @@ do
     /app/bin/bitcoin-cli -regtest generatetoaddress 1 $ADDRESS
     sleep 5
     # send tx to own address while paying fee to the miner
-    /app/bin/bitcoin-cli -regtest sendtoaddress $ADDRESS 1 "" "" true
+    /app/bin/bitcoin-cli -regtest -named sendtoaddress address=$ADDRESS amount=0.1 subtractfeefromamount=false fee_rate=1
```

Comment:
```suggestion
    /app/bin/bitcoin-cli -regtest -named sendtoaddress address=$ADDRESS amount=0.1 subtractfeefromamount=true fee_rate=1
```
This needs to be true